myriad-scheduler/src/main/java/com/ebay/myriad/state/SchedulerState.java

```diff
@@ -42,16 +48,18 @@
     private Set<Protos.TaskID> activeTasks;
     private Set<Protos.TaskID> lostTasks;
     private Set<Protos.TaskID> killableTasks;
-    private MyriadState myriadState;
+    private Protos.FrameworkID frameworkId;
+    private MyriadStateStore stateStore;
 
-    public SchedulerState(MyriadState myriadState) {
```

Comment:
Cleanup: Now that MyriadState is not used, can MyriadState.java be removed? Also, ```MyriadModule.providesSchedulerState``` and ```MesosModule.providesStateStore``` can also be deleted I guess.